.circleci/config.yml

```diff
@@ -292,6 +291,34 @@ jobs:
           qualifier: "<< parameters.qualifier >>"
           releaseChannel: "<< parameters.releaseChannel >>"
 
+  clean_up_dev_deployments:
+    executor:
+      name: hmpps/node
+      tag: << pipeline.parameters.node-version >>
+    steps:
+      - checkout
+      - hmpps/k8s_setup:
+          ca-cert: KUBE_CLUSTER_CERT
+          cluster-name: KUBE_CLUSTER_NAME
+          kube-namespace: KUBE_NAMESPACE
+          kube-token: KUBE_TOKEN
+      - hmpps/install_helm
+      - run:
+          name: Clean up dev deployments
+          command: |
+            curl -s "https://api.github.com/repos/ministryofjustice/prisoner-content-hub-frontend/pulls" | jq  '.[].number' | sort > prs.txt
+            helm --namespace prisoner-content-hub-development list | awk '{ print $1 }' | tail -n +2 | grep 'frontend-' | sed "s/[^0-9]//g" | sort >> deployments.txt
```

Comment:
* list currently deployed releases in dev
* print out the first column (release name)
* drop the header row (technically unnecessary I guess) 
* Filter out all releases that are not versioned frontend ones
* Extract out just the version qualifier
* Sort
* Write to file

Reply:
Yeah... the `tail -n +2` serves no purpose. 
Also the sort line _>>_ appends to the file? Makes no difference if it doesn't already exist?

Reply:
didn't mean to `>>` and I'll remove the `tail -n`